src/nl/rubensten/texifyidea/inspections/latex/LatexFileNotFoundInspection.kt

```diff
@@ -35,20 +36,30 @@ open class LatexFileNotFoundInspection : TexifyInspectionBase() {
         for (command in commands) {
             // Only consider default commands with a file argument.
             val default = LatexCommand.lookup(command.name) ?: continue
-            val arguments = default.arguments
-            val parameters = command.parameterList
+            // Remove optional arguments from list of commands
+            val arguments = default.arguments.filter { it is RequiredArgument }
+                    .map { it as RequiredArgument }
```

Comment:
```kotlin
default.arguments.mapNotNull { it as? RequiredArgument }
```

Reply:
this improvment is really cool. This is the first project in Kotlin, so I am learning new things all the time. I will add this.
(I like Kotlin for these little sweets like this.)

Reply:
It's also more performant on larger sets as `filter` creates a new collection, and `map` creates a new collection. Especially when more operations are chained, this can lead to big performance drops. So always make sure to use sequences whenever you need more than 3 operations (not necessarily so for collections of only a handful of elements).